### DISCUSSION TITLE
BasicForm 怎么自由控制每行放置的input数量

### DISCUSSION THREAD

**Discussion by bieshaoxiong** [vbenjs/vue-vben-admin#3177] [General]
![image](https://user-images.githubusercontent.com/64937534/201466294-89013492-b371-4bb7-9087-16d30e621adc.png)
为了布局稍微好看点，我这里想空着，换行继续防止input，现在是用 Divider 占着位置，但是有点丑，
有什么方法能设置换行吗

**Comment by gh852195168** [comment]
自定义渲染个空的, 或者用插槽
